packages/@aws-cdk/aws-eks/lib/managed-nodegroup.ts

```diff
@@ -443,24 +456,40 @@ export class Nodegroup extends Resource implements INodegroup {
   }
 }
 
+const arm64AmiTypes: NodegroupAmiType[] = [NodegroupAmiType.AL2_ARM_64, NodegroupAmiType.BOTTLEROCKET_ARM_64];
+const x8664AmiTypes: NodegroupAmiType[] = [NodegroupAmiType.AL2_X86_64, NodegroupAmiType.BOTTLEROCKET_X86_64];
+const gpuAmiTypes: NodegroupAmiType[] = [NodegroupAmiType.AL2_X86_64_GPU];
+
 function getAmiTypeForInstanceType(instanceType: InstanceType) {
```

Comment:
1. suggest to rename the function to `getAmiTypesForInstanceType` or `getPossibleAmiTypesForInstanceType`.
2. can specify return type on function declaration

Reply:
Thanks! Just fixed.